RELEASES.md

```diff
@@ -1,3 +1,117 @@
+- [`AtomicI16::from_ptr`]
+- [`AtomicI32::from_ptr`]
+- [`AtomicI64::from_ptr`]
+- [`AtomicIsize::from_ptr`]
+- [`<ptr>::is_null`]
+- [`<ptr>::as_ref`]
+- [`<ptr>::as_mut`]
+- [`core::ptr::without_provenance`]
+- [`core::ptr::without_provenance_mut`]
+- [`core::ptr::dangling`]
+- [`core::ptr::dangling_mut`]
+- [`Pin::new`]
+- [`Pin::new_unchecked`]
+- [`Pin::get_ref`]
+- [`Pin::into_ref`]
+- [`Pin::get_mut`]
+- [`Pin::get_unchecked_mut`]
+- [`Pin::static_ref`]
+- [`Pin::static_mut`]
+- [`<int>::isqrt`]
+- [`<int>::checked_isqrt`]
+- [`<uint>::isqrt`]
+- [`NonZero::isqrt`]
+
+<a id="1.84.0-Cargo"></a>
+
+Cargo
+-----
+- [Stabilize MSRV-aware resolver config](https://github.com/rust-lang/cargo/pull/14639/)
+- [Stabilize resolver v3](https://github.com/rust-lang/cargo/pull/14754/)
+
+<a id="1.84-Rustdoc"></a>
+
+Rustdoc
+-------
+
+- [rustdoc-search: improve type-driven search](https://github.com/rust-lang/rust/pull/127589)
+
+<a id="1.84.0-Compatibility-Notes"></a>
+
+Compatibility Notes
+-------------------
+- [Enable by default the `LSX` target feature for LoongArch Linux targets](https://github.com/rust-lang/rust/pull/132140)
+- [The unstable `-Zprofile` flag (“gcov-style” coverage instrumentation) has been removed.](https://github.com/rust-lang/rust/pull/131829) This does not affect the stable flags for coverage instrumentation (`-Cinstrument-coverage`) and profile-guided optimization (`-Cprofile-generate`, `-Cprofile-use`), which are unrelated and remain available.
+- Support for the target named `wasm32-wasi` has been removed as the target is now named `wasm32-wasip1`. This completes the [transition](https://github.com/rust-lang/compiler-team/issues/607) [plan](https://github.com/rust-lang/compiler-team/issues/695) for this target following [the introduction of `wasm32-wasip1`](https://github.com/rust-lang/rust/pull/120468) in Rust 1.78. Compiler warnings on [use of `wasm32-wasi`](https://github.com/rust-lang/rust/pull/126662) introduced in Rust 1.81 are now gone as well as the target is removed.
+<a id="1.84.0-Internal-Changes"></a>
+- [The syntax `&pin (mut|const) T` is now parsed as a type which in theory could effect macro expansion results in some edge cases](https://github.com/rust-lang/rust/pull/130635#issuecomment-2375462821)
+- [Legacy const generic arguments are no longer permitted to declare items or bodies (such as closures, inline consts, or async blocks). This does not effect normal usages of const generics in any way.](https://github.com/rust-lang/rust/pull/130443#issuecomment-2445678945)
+- The `wasm32-unknown-emscripten` target's binary release of the standard library is now [built with the latest emsdk 3.1.68](https://github.com/rust-lang/rust/pull/131533), which fixes an ABI-incompatibility with Emscripten >= 3.1.42. If you are locally using a version of emsdk with an incompatible ABI (e.g. before 3.1.42 or a future one), you should build your code with `-Zbuild-std` to ensure that `std` uses the correct ABI.
```

Comment:
and again compat notes (i guess the anchor tag above was misplaced)

Reply:
According to [this comment](https://github.com/rust-lang/rust/issues/131467#issuecomment-2529314603), the Emscripten version is still the same as before.